newm-server/src/main/kotlin/io/newm/server/features/cardano/repo/CardanoRepository.kt

```diff
@@ -50,6 +50,11 @@ interface CardanoRepository {
 
     suspend fun queryLiveUtxos(address: String): List<Utxo>
 
+    suspend fun queryUtxosByOutputRef(
```

Comment:
We can remove this. We already get the addresses with the input utxos from the cip30 wallet. This is only for instances where we don't actually have addresses with a utxo.

Reply:
Done